apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -21,6 +21,7 @@ import {
 } from "../../locales/client";
 import { SaveAction } from "../../lib/api/external/laskugeneraattori/actions";
 import { type InvoiceGeneratorFormState } from "../../lib/api/external/laskugeneraattori/index";
+const MAX_PAYLOAD_SIZE = 24 * 1024 * 1024; // 24MB in bytes
```

Comment:
This is good for now, but it also couples this limit with the backend. Alternative would be to handle the 413 error gracefully and then show the limit returned by the server.

But it's definitely better usability to show it in advance.